src/app/(application)/meus-cupons/page.tsx

```diff
@@ -16,26 +21,34 @@ import Arrow from "@/images/easytolive/icons/arrow-next-right-primary.svg";
 interface IfilterOptions {
   id: string;
   name: string;
+  emptyText: string;
   icon: StaticImageData;
   textColor: string;
+  href?: string;
+  HrefLabel?: string;
```

Comment:
```suggestion
  hrefLabel?: string;
```

why use "H" upper in this case? 
Let's follow a simple pattern for all props.